range_tombstone_list.cc

```diff
@@ -306,78 +306,56 @@ range_tombstone_list::reverter range_tombstone_list::apply_reversibly(const sche
     return rev;
 }
 
+struct bv_order_by_end {
```

Comment:
Would it make sense to hide those structs in anonymous namespace?

Reply:
It does - I changed it.